api/cloudcontroller/ccv3/internal/api_routes.go

```diff
@@ -48,6 +49,7 @@ const (
 	PostIsolationSegmentRelationshipOrganizationsRequest    = "PostIsolationSegmentRelationshipOrganizations"
 	PostIsolationSegmentsRequest                            = "PostIsolationSegments"
 	PostPackageRequest                                      = "PostPackageRequest"
+	PostServiceInstanceRelationshipSharedSpacesRequest      = "PostServiceInstanceRelationshipSharedSpaces"
```

Comment:
please add a "s" to "Relationship" ("Relationships") because our naming convention follows the CC endpoint name

Reply:
👍